metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -97,6 +101,61 @@ class GEProfilerRequest:
     batch_kwargs: dict
 
 
+def get_column_unique_count_patch(self, column):
+    if self.engine.dialect.name.lower() == "redshift":
+        element_values = self.engine.execute(
+            f"SELECT APPROXIMATE count (distinct {column}) FROM {self._table}"
+        )
+        return convert_to_json_serializable(element_values.fetchone()[0])
+    elif self.engine.dialect.name.lower() == "bigquery":
+        element_values = self.engine.execute(
+            f"SELECT APPROX_COUNT_DISTINCT ({column}) FROM {self._table}"
+        )
+        return convert_to_json_serializable(element_values.fetchone()[0])
+    elif self.engine.dialect.name.lower() == "snowflake":
+        element_values = self.engine.execute(
+            f"SELECT APPROX_COUNT_DISTINCT ({column}) FROM {self._table}"
+        )
+        return convert_to_json_serializable(element_values.fetchone()[0])
+    return convert_to_json_serializable(
+        self.engine.execute(
+            sa.select([sa.func.count(sa.func.distinct(sa.column(column)))]).select_from(
+                self._table
+            )
+        ).scalar()
+    )
+
+
+def _get_column_quantiles_bigquery_patch(  # type:ignore
+    self, column: str, quantiles: Iterable
+) -> list:
+    quantile_queries = list()
+    for quantile in quantiles:
+        quantile_queries.append(
+            f"approx_quantiles({column}, 100) OFFSET [{quantile * 100}]"
+        )
+
+    select_part = ",".join(quantile_queries)
```

Comment:
once again, I'd rather that we use SQLAlchemy for some of these operations.

Reply:
As we discussed I rewrote it with SQLAlchemy